chef/cookbooks/provisioner/templates/default/autoyast.xml.erb

```diff
@@ -27,6 +27,13 @@
     </signature-handling>
     <storage/>
   </general>
+  <bootloader>
```

Comment:
@dirkmueller This is the same for SLE11/12. IMO if section would be overkill here.